parameters/src/testnet1/mod.rs

```diff
@@ -51,14 +51,14 @@ impl_params_remote!(
     "https://s3-us-west-1.amazonaws.com/aleo.parameters",
     "./",
     "inner_snark_pk",
-    190452497
+    189389633
 );
 impl_params_local!(
     InnerSNARKVKParameters,
     inner_snark_vk_test,
     "./",
     "inner_snark_vk",
-    2426
+    2232
```

Comment:
Note: Using `SchnorrCompressed` reduces the size of input to the inner circuit and outer circuit, so the size of the VK goes down.

Reply:
@weikengchen How confident are you that these changes to VK size of inner and outer are correct?

Upon merging with `testnet2` (post #348), it seems the original sizes are correct, and I confirmed that SchnorrCompressed is being used in DPC circuits

Reply:
I think I also need to double check---it is expected that the VK should be smaller because there are one fewer element in the input to the inner circuit.